docs/usage/getting-started/private-packages.md

```diff
@@ -154,7 +154,11 @@ It can be confusing for people who host their own source code privately to be as
 Currently the preferred way to configure `github.com` credentials for self-hosted Renovate is:
 
 - Create a read-only Personal Access Token (PAT) for a `github.com` account. This can be any GitHub account, but we recommend you create an "empty" account for this purpose.
-- Add the PAT to Renovate using the environment variable `GITHUB_COM_TOKEN`
+- Add the PAT to Renovate using the environment variable `RENOVATE_GITHUB_COM_TOKEN`
+
+<!-- prettier-ignore -->
+!!! note
+    `GITHUB_COM_TOKEN` is still parsed and takes precedence over `RENOVATE_GITHUB_COM_TOKEN`, but is considered deprecated and will be removed in a future major update.
```

Comment:
shouldn't we do it the other way around? 🤔 we do it for log variables

Reply:
My idea has been that if somebody has this variable set for whatever reason, maybe because the person has expected the prefix, then it would break. So this would be technically a breaking change. 

With this we give users the possibility to migrate and then in v40 we can remove support for it.

Reply:
Sure, let's change precedence in v40